README.rst

```diff
@@ -33,38 +33,78 @@ RENKU (連句)
+environment specification. Every result can be replayed either to repeat a
+calculation or to re-execute on new data or with a different choice of
+parameters.
 
-**The Renku Project** provides a platform and tools for reproducible and
-collaborative data analysis projects. It is aimed at independent researchers and
-data scientists as well as labs, collaborations, and courses and workshops.
-Renku can be used by anyone who deals with data, whether they are a researcher,
-data analyst, project owner, or data provider.
+Renku encourages **reusability** by storing and querying the connections between
+datasets, code executions, and results in a Knowledge Graph. Producers and
+consumers of analysis artifacts can always recover the full provenance of a
+result, establishing trust and reducing boilerplate.
 
-It consists of:
+Renku stimulates **collaboration** among peers and across disciplines by
+guaranteeing that a media-rich discussion space and fully configured, shareable
+interactive computational environments are always just a click away.
+Collaborators can easily work on projects together or in parallel, combining
+their work in a systematic and safe manner.
 
-* `Renkulab <https://renku.readthedocs.io/en/latest/introduction/renkulab.html#renkulab>`_,
-  a web-based platform designed to facilitate collaboration, reproducibility, and discovery.
 
-* `Renku <https://renku.readthedocs.io/en/latest/introduction/renku.html#renku>`_,
-  a command-line interface (CLI) (`docs <https://renku-python.readthedocs.io/en/latest/>`_,
-  `repo <https://github.com/SwissDataScienceCenter/renku-python>`_) for managing code, datasets,
-  and workflows that you can use on Renkulab or install locally.
+Getting Started
+---------------
 
-A public instance of **Renkulab** is available at https://renkulab.io, and several
-other deployments at various institutions. Go ahead - log in, try it out, and
-`let us know what you think <http://bit.ly/renku-feedback>`_! You can follow the
-`first_steps <https://renku.readthedocs.io/en/latest/tutorials/01_firststeps.html>`_ tutorial in python or R or `continue reading about the
-Renku project <https://renku.readthedocs.io/en/latest/introduction/index.html#renku-introduction>`_.
+The Renku platform consists of `RenkuLab
+<https://renku.readthedocs.io/en/latest/introduction/renkulab.html#renkulab>`_,
+a web-based application and `Renku
+<https://renku.readthedocs.io/en/latest/introduction/renku.html#renku>`_, a
+command-line tool for managing code, data, workflows and making practical use of
+the Knowledge Graph.
 
+A public instance of **RenkuLab** is available at https://renkulab.io, and there
+are several other deployments at various institutions. To start exploring Renku,
+feel free to make an account try it out! You can follow the `first steps
+<https://renku.readthedocs.io/en/latest/tutorials/01_firststeps.html>`_ tutorial
+in python or R or `continue reading about the Renku project
```

Comment:
Though I think we should have R and Julia versions of the first-steps tutorial, we don't actually have anything except Python at the moment. So we can just say 

```
You can follow the `first steps
 <https://renku.readthedocs.io/en/latest/tutorials/01_firststeps.html>`_ tutorial
 or `continue reading about the Renku project...
```

Reply:
oops true, I don't know when R was added there...